### PR TITLE
Revise Search Operations section

### PR DESCRIPTION
**Pull request by jamesm-macrometa**
This is the start of my revisions for Search Operations section.  The later sections need work, but what's been edited is tested and ready for review.

### CODE REVIEW

**Review by martin-macrometa — APPROVED** [review]

File: docs/search/operations.md [hunk 1]
```diff
@@ -3,34 +3,31 @@ sidebar_position: 40
 title: Search Queries
 ---
 
-The `SEARCH` keyword starts the language construct to filter Views of type Search. Conceptually, a View is just another document data source, similar to an array or a document/edge collection, over which you can iterate using a [FOR operation](../queries/c8ql/operations/for.md) in C8QL:
+You can use the `SEARCH` keyword to filter a query using a search view, providing these features:
```
Comment (diana-macrometa) [inline docs/search/operations.md]:
```suggestion
You can use the `SEARCH` keyword to filter a query using a search view, allowing you to:
```

File: docs/search/operations.md [hunk 2]
```diff
@@ -3,34 +3,31 @@ sidebar_position: 40
...
 
-```sql
-FOR doc IN viewName
-  RETURN doc
-```
-
-The optional `SEARCH` operation provides the capabilities to:
+- Filter documents based on C8QL Boolean expressions and functions.
```
Comment (diana-macrometa) [inline docs/search/operations.md]: C8QL and SQL?

File: docs/search/operations.md [hunk 3]
```diff
@@ -3,34 +3,31 @@ sidebar_position: 40
...
+- Match documents located in different collections.
+- Sort the result set based on how closely each document matched the search conditions.
```
Comment (diana-macrometa) [inline docs/search/operations.md]:
```suggestion
- Sort the result set based on how closely each document matches the search conditions.
```

File: docs/search/operations.md [hunk 4]
```diff
@@ -3,34 +3,31 @@ sidebar_position: 40
...
-- match documents located in different collections backed by a fast index
-- sort the result set based on how closely each document matched the search conditions
+Refer to [Search Views](/search/views/index.md) for information about creating a custom search view.
 
-See [Search Views](../index.md) on how to set up a View.
-
-## General Syntax
-
-The `SEARCH` keyword is followed by an Search filter expressions, which is mostly comprised of calls to Search C8QL functions.
+You must use the `SEARCH` statement in a `FOR...IN` operation. The `SEARCH` statement must be placed after the `FOR` and before any additional operations.
```
Comment (diana-macrometa) [inline docs/search/operations.md:16]:
```suggestion
## Syntax

You must use the `SEARCH` statement in a `FOR...IN` operation. The `SEARCH` statement must be placed after the `FOR` and before any additional operations.
```

File: docs/search/operations.md [hunk 5]
```diff
@@ -43,78 +40,56 @@ The `SEARCH` statement, in contrast to `FILTER`, is treated as a part of the `FO
...
-:::warning
-The alphabetical order of characters is not taken into account by Search, i.e. range queries in SEARCH operations against Views will not follow the language rules as per the defined Analyzer locale.
-:::
-
-```sql
-FOR doc IN viewName
-  SEARCH ANALYZER(doc.text == "quick" OR doc.text == "brown", "text_en")
-RETURN doc
-```
+`SEARCH` does not support:
```
Comment (diana-macrometa) [inline docs/search/operations.md:47]:
```suggestion
## Limitations

`SEARCH` does not support:
```

File: docs/search/operations.md [hunk 6]
```diff
@@ -43,78 +40,56 @@ The `SEARCH` statement, in contrast to `FILTER`, is treated as a part of the `FO
...
+- In-line expressions
 
-The `OPTIONS` keyword and an object can optionally follow the search expression to set [Search Options](#search-options).
+Refer to [Search Options](#search-options) for information about the `OPTIONS` keyword.
 
-## Handling of non-indexed fields
+## Search by Document Attribute
 
-Document attributes which are not configured to be indexed by a View are treated by `SEARCH` as non-existent. This affects tests against the documents emitted from the View only.
+You can search for documents by querying document attributes that have been indexed by both the search view and the document store. Refer to [Document Store Indexes](/collections/documents/document-store-indexes.md) for more information about adding attributes to document store indexes.
```
Comment (diana-macrometa) [inline docs/search/operations.md]: Test this link, it looks wrong.

File: docs/search/operations.md [hunk 7]
```diff
@@ -186,6 +161,8 @@ With `trackListPositions: true` you would need to specify the index of the array
 ANALYZER(doc.text[2] == 'jump', "text_en")
 ```
 
+
+
```
Comment (diana-macrometa) [inline docs/search/operations.md]: Remove extra empty lines.

File: docs/search/operations.md [hunk 8]
```diff
@@ -186,6 +161,8 @@ With `trackListPositions: true` you would need to specify the index of the array
...
 ## SEARCH with SORT
 
 The documents emitted from a View can be sorted by attribute values with the standard [SORT() operation](../queries/c8ql/operations/sort.md), using one or multiple attributes, in ascending or descending order (or a mix thereof).
```
Comment (diana-macrometa) [inline docs/search/operations.md]:
```suggestion
The documents emitted from a search view can be sorted by attribute values with the standard [SORT() operation](../queries/c8ql/operations/sort.md), using one or multiple attributes, in ascending or descending order (or a mix thereof).
```

File: docs/search/operations.md [hunk 9]
```diff
@@ -221,8 +198,6 @@ The `SEARCH` operation accepts an options object with the following attributes:
 
 - `collections` (array, _optional_): array of strings with collection names to restrict the search to certain source collections
 
-**Examples**
-
 Given a View with three linked collections `coll1`, `coll2` and `coll3` it is possible to return documents from the first two collections only and ignore the third using the `collections` option:
```
Comment (diana-macrometa) [inline docs/search/operations.md]:
```suggestion
Given a search view with three linked collections (`coll1`, `coll2`, and `coll3`), you can return documents from the first two collections only and ignore the third using the `collections` option:
```

**Review by diana-macrometa — APPROVED** [review]

File: docs/search/operations.md [hunk 10]
```diff
@@ -3,34 +3,33 @@ sidebar_position: 40
...
+For example:
+
+```sql
+FOR doc IN MySearchView
+  SEARCH ANALYZER(doc.text == "quick" OR doc.text == "brown", "text_en")
+RETURN doc
+```
 
-`expression` must be an Search expression. The full power of Search is harnessed and exposed via special [Search functions](functions.md), during both the search and sort stages. On top of that, common C8QL operators are supported:
+Replace `SEARCH_VIEW_NAME` with the name of your search view, and `EXPRESSION` with one of the following [Search functions](functions.md):
```
Comment (Luba89) [inline docs/search/operations.md:33]: AND, OR ... are logical and comparison operators; they are not Search functions. PHRASE STARTSWITH() ... are Search functions

**Pull request merged**